web/cobrands/fixmystreet.com/layout.scss

```diff
@@ -156,29 +156,69 @@ body.fullwidthpage {
 }
 
 body.frontpage {
+    $fms-logo-width: 200px;
```

Comment:
Again, this feels very large to me, especially given we’ve reduced the size of the FMS logo at the same time.